library/alert/alert.css

```diff
@@ -0,0 +1,63 @@
+.alert {
+    display: flex;
+    width: 48rem;
+    height: 8rem;
+    padding: 0.8rem 1.6rem;
+    background: rgba(var(--rgba-alert-success-light-background-color));
+}
+
+.alert__icon {
+    width: 4.8rem;
+    height: 4.8rem;
+}
+
+.alert__content {
+    padding: 0rem 0.8rem;
+    border-radius: 0.4rem;
+}
+
+.alert__text {
+    color: var(--alert-success-light-font-color);
+}
+
+/* Alert Component Starts -- Success */
+.alert--success {
+    background: rgba(var(--rgba-alert-success-light-background-color));
+}
+
+.alert__text--success {
+    color: var(--alert-success-light-font-color);
+}
+/* Alert Component Ends -- Success */
+
+/* Alert Component Starts -- Error */
+.alert--error {
+    background: rgba(var(--rgba-alert-error-light-background-color));
+}
+
+.alert__text--error {
+    color: var(--alert-error-light-font-color);
+}
+/* Alert Component Ends -- Error */
+
+
+/* Alert Component Starts -- Warning */
+.alert--warning {
+    background: rgba(var(--rgba-alert-warning-light-background-color));
```

Comment:
why rgba here?

Reply:
To provide alpha value, Just checked and realized we can use alpha values in hex as well. 
Will be refactoring the code accordingly.

Thanks for pointing it out.